templates/todo/projects/nodejs-mongo/repo-bicep.yaml

```diff
@@ -58,3 +69,11 @@ repo:
       ignore:
         - "build/**/*"
         - "node_modules/**/*"
+
+    # Infra 
+    - from: ./infra/bicep
+      to: ./infra
+
+    # Azure.yml
+    - from: ./azure-bicep.yaml
```

Comment:
Why renamed azure.yaml to azure-bicep.yaml?

Reply:
in the proposed design, Every projec within the todo folder can have multiple azure-<IACTOOL>.yml, and repo-<IACTOOL>.yml. along with a sub folder by IAC name within infra . This give a path to adding the multi IAC tool support and deploying to different iac sample templates with out duplicating projects within todo folder 
![image](https://user-images.githubusercontent.com/34171319/185668845-8ded94fc-fdd1-4511-a008-2e34f02c0a40.png)